### PR TITLE
Fix team names and descriptions

### PR DESCRIPTION
**Pull request by LEARAX**
The grammar of team names and descriptions on the governance page of rust-lang.org is very inconsistent. This PR attempts to correct this, by changing each description to start with a present progressive verb, applying consistent capitalization rules, and ending most of the team names with either "working group" or "team" (moderator groups and observers were left as exceptions).

### CODE REVIEW

**Comment by LEARAX** [comment]
This would address #31.

**Comment by LEARAX** [comment]
In #124 the new WG description was terminated with a period. Is this syntax preferred to no periods?

**Comment by LEARAX** [comment]
I'm unsure if I need to ping people to review this. @pietroalbini

**Comment by pietroalbini** [comment]
cc @rust-lang/core 
I shouldn't be the one reviewing this.

**Comment by nikomatsakis** [comment]
I can do an initial review.

**Comment by vertexclique** [comment]
Please also take a look at this description for `wg-localization`. I changed this while I was adding myself, but can be discarded.
https://github.com/rust-lang/team/pull/180/commits/49e984242066b0d89955709435e2109c7bf6ae1c

File: teams/devtools.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ ping = "rust-lang/dev-tools"
 page = "dev-tools"
 name = "Dev tools team"
 email = "tools@rust-lang.org"
-description = "Rust developer tools"
+description = "Contributing to and creating the Rust develpoment tools"
```
Comment (Mark-Simulacrum) [inline teams/devtools.toml]:
```suggestion
description = "Contributing to and creating the Rust development tools"
```

File: teams/release.toml [hunk 2]
```diff
@@ -30,7 +30,7 @@ ping = "rust-lang/release"
 
 [website]
 name = "Release team"
-description = "tracking regressions, stabilizations, and producing Rust releases"
+description = "Tracking regressions, developing stabilizations, and producing Rust releases"
```
Comment (Mark-Simulacrum) [inline teams/release.toml]:
```suggestion
description = "Tracking regressions and stabilizations, and producing Rust releases"
```

**Comment by Mark-Simulacrum** [comment]
Looks like there's a bunch of merge conflicts, but this looks good to me modulo the few commits I left. Could you rebase it?

**Comment by LEARAX** [comment]
Never rebased before. Hopefully that worked.

File: teams/wg-async-await.toml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+wg = true
+
+[people]
+leads = ["nikomatsakis", "cramertj"]
+members = ["nikomatsakis", "cramertj"]
+
+[website]
+name = "Async-await implementation working group"
+description = "Implementing async-await functionality"
+repo = "https://rust-lang.github.io/compiler-team/working-groups/async-await/"
```
Comment (Mark-Simulacrum) [inline teams/wg-async-await.toml]: Looks like a rebase bug? This file should probably just not be added.

**Comment by Mark-Simulacrum** [comment]
By the way, I plan to squash this all into one commit when merging (or you can do so on your branch), I don't think we need a commit per file.

File: teams/wg-async-await.toml (hunk 3 above)
Comment (LEARAX) [reply teams/wg-async-await.toml]: Guess it slipped past. Fixed now.

**Comment by LEARAX** [comment]
@Mark-Simulacrum Probably not, there are quite a few commits.

**Comment by Mark-Simulacrum** [comment]
Thanks!

**Comment by RalfJung** [comment]
Should this have closed https://github.com/rust-lang/team/issues/31 ?

**Pull request merged**